<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyMessenger</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/icons8-parrot.png</normaloff>../../../Downloads/icons8-parrot.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="textEditUsername">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddContact">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Downloads/icons8-plus.png</normaloff>../../../Downloads/icons8-plus.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDelContect">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Del</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Downloads/if_f-cross_256_282471.ico</normaloff>../../../Downloads/if_f-cross_256_282471.ico</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Контакты</string>
    </property>
    <widget class="QListWidget" name="listWidgetContacts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidgetMessages">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonSend">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>285</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonActionBold">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>D:/Download_firefox/selectedformat/selectedformat/b.jpg</normaloff>D:/Download_firefox/selectedformat/selectedformat/b.jpg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonActionItalic">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>D:/Download_firefox/selectedformat/selectedformat/i.jpg</normaloff>D:/Download_firefox/selectedformat/selectedformat/i.jpg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButtonActionUnderlined">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>D:/Download_firefox/selectedformat/selectedformat/u.jpg</normaloff>D:/Download_firefox/selectedformat/selectedformat/u.jpg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTextEdit" name="textEditMessage"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
